prow/test/integration/integration-test.sh

```diff
@@ -33,6 +33,7 @@ function retry() {
 function setup() {
   "${bazel}" run //prow/test/integration:setup-local-registry "$@" || ( echo "FAILED: set up local registry">&2; return 1 )
 
+  PUSH=true KO_DOCKER_REPO="localhost:5001" ./hack/make-rules/prow-images.sh
```

Comment:
So the integration test setup now publishes images with ko then overwrites them by publishing with bazel? Is that what is intended?

Reply:
yeah at this stage it's intentional to run images building with both and let images built from bazel overriding the other, as they are what's used in production right now